src/aiovodafone/api.py

```diff
@@ -264,6 +274,14 @@ async def get_all_devices(self) -> dict[str, VodafoneStationDevice]:
 
         return self._devices
 
+    async def convert_uptime(self, uptime: str) -> datetime:
+        """Convert router uptime to last boot datetime."""
+        d = int(uptime.split(":")[0])
+        h = int(uptime.split(":")[1])
+        m = int(uptime.split(":")[2])
+
+        return datetime.utcnow() - timedelta(days=d, hours=h, minutes=m)
```

Comment:
You might be able to use `isodata.parse_duration` here .. but what you have is probably fine for this usecase